### ISSUE TITLE
Tutorial page 6 fails at `meteor install-sdk android`

### ISSUE THREAD

**Issue by torbengb** [meteor/tutorials#168]
I'm following the "simple-todo tutorial and arrive at page 6 https://www.meteor.com/tutorials/blaze/running-on-mobile which says:

> **Running on an Android emulator**
In the terminal, go to your app folder and type:
`meteor install-sdk android`
This will help you install all of the necessary tools to build an Android app from your project. When you are done installing everything, type:

However, running the command `meteor install-sdk android` produces this output:
> Please follow the installation instructions in the mobile guide:
http://guide.meteor.com/mobile.html#installing-prerequisites

And that takes me to a long list of new instructions. Clearly, it's not as simple as the tutorial page 6 makes it out to be. 

----
Meteor 1.6.1  
Ubuntu 16.04 LTS

**Comment by MichaelJCole** [comment]
@torbengb Did you find a better way to do this?  This is installing other people's software so it's outside the scope of a tutorial.  Can we close this?

**Comment by torbengb** [comment]
Yeah, please close it. I never got to fix this, and eventually abandoned
it.
